### PR TITLE
`gpaa-store-static-map-entry-meta.php`: Added snippet to save static map in Gravity Forms Entry Meta.

### PR DESCRIPTION
**Pull request by saifsultanc**
## Context

⛑️ Ticket(s): https://secure.helpscout.net/conversation/2615434629/67102

## Summary

Storing Static Map via Google Maps Static API on the entry meta, at the time of form submission.

### CODE REVIEW

File: gp-address-autocomplete/gpaa-store-static-map-entry-meta.php [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+	// Target Form ID, and the Address (not Map) Field ID.
+	$form_id  = '310';
+	$field_id = '3';
+
+	// Values for Static Map Setting
+	$zoom    = 13;
+	$width   = 640;
+	$height  = 480;
+	$api_key = 'Enter your Google Maps API Key';
```
Comment (claygriffiths) [inline gp-address-autocomplete/gpaa-store-static-map-entry-meta.php:25]: Be sure to address `$api_key`, `$form_id`, and `$field_id` in the instructions

File: gp-address-autocomplete/gpaa-store-static-map-entry-meta.php [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+	if ( $form['id'] != $form_id ) {
+		return;
+	}
+
+	$latitude  = $entry[ 'gpaa_lat_' . $field_id ];
+	$longitude = $entry[ 'gpaa_lng_' . $field_id ];
+	$url       = "https://maps.googleapis.com/maps/api/staticmap?center={$latitude},{$longitude}&zoom={$zoom}&size={$width}x{$height}&maptype=roadmap&key={$api_key}";
+
+	gform_update_meta( $entry['id'], 'gppa_' . $field_id, $url );
```
Comment (claygriffiths) [inline gp-address-autocomplete/gpaa-store-static-map-entry-meta.php:35]: Curious, why this meta key? Should it be slightly more specific such as `'gppa_' . $field_id . '_map'`?

Also, is it possible to display this in the entry detail view?

Comment (saifsultanc) [reply gp-address-autocomplete/gpaa-store-static-map-entry-meta.php:35]: Updated customer with a class based gist snippet, which includes adding the entry meta box. Perhaps we can push that as an overall update here on snippet-library after customer affirms?

**Comment by saifsultanc** [comment]
Closing in favour of https://github.com/gravitywiz/snippet-library/pull/840

**Pull request closed without merging**